src/Integration/Integration.csproj

```diff
@@ -256,11 +257,17 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```

Comment:
Not really but it will add it each time you sync, we will not be packaging it into the vsix so instead of undoing it each time I decided to save it and avoid the pain in the future